cli/src/templates/packages/expo-router/stack/app/+not-found.tsx.ejs

```diff
@@ -114,12 +111,12 @@ export default function NotFoundScreen() {
       fontSize: 14,
       color: '#2e78b7',
     },
-  });
+}));
```

Comment:
An extra parentheses was added here which was breaking the tests. I will push a fix to remove. Just leaving this note to let you know what I changed.
```suggestion
});
```

Reply:
Ah right, wrong template. It's for `StyleSheet`. Thanks for spotting it out